docs/index.md

```diff
@@ -26,17 +26,72 @@ First [install](install_guide) the package.
+  - Wichmann, F. A. and Hill, N. J. (2001). [The psychometric function: I. Fitting, sampling and goodness-of-fit](https://link.springer.com/article/10.3758/BF03194544). *Perception and Psychophysics*, 63(8), 1293-1313.
+  - Wichmann, F. A. and Hill, N. J. (2001). [The psychometric function: II. Bootstrap-based confidence intervals and sampling](https://link.springer.com/article/10.3758/BF03194545). *Perception and Psychophysics*, 63(8), 1314-1329.
 
-We would like to thank previous members of the Wichmann-lab who
-were involved in developing MCMC based methods of Bayesian inference for
-the psychometric function, most notably Ingo Fründ, Valentin Hänel,
-Frank Jäkel and Malte Kuss.
+A copy of the original website and code is still [available](https://uni-tuebingen.de/fakultaeten/mathematisch-naturwissenschaftliche-fakultaet/fachbereiche/informatik/lehrstuehle/neuronale-informationsverarbeitung/research/software/psignifit-legacy/).
 
-Furthermore, our thanks go to the reviewers of our manuscript and the
+*psignifit* `3` was developed by Ingo Fründ, Valentin Hänel and Felix Wichmann at the
+TU Berlin between 2009 and 2012 using a mix of Python and C/C++, inspired by Malte Kuss, Frank Jäkel and
+Felix Wichmann's Bayesian psychometric function fitting package PsychoFun written in R:
+
+  - Kuss, M., Jäkel, F. and Wichmann, F. A. (2005). [Bayesian inference for psychometric functions](http://jov.arvojournals.org/article.aspx?articleid=2192844). *Journal of Vision*, 5, 478-492
+  - Fründ, I., Haenel, N. V. and Wichmann, F.A. (2011). [Inference for psychometric functions in the presence of nonstationary behavior](http://jov.arvojournals.org/article.aspx?articleid=2121082). *Journal of Vision*, 11(6:16), 1-19.
+
+The goal of the re-write was twofold: First, to move from ML parameter
+estimation and bootstrapping to Bayesian inference. Second, to move from MATLAB
+to Python as programming language. However, it is important to note that the
+MCMC sampling used for Bayesian inference in *psignifit* `3` is not always reliable,
+and further development of *psignifit* `3` was stopped. The original website is still
+[online](https://psignifit.sourceforge.net/) though.
+
+
+For the latest *psignifit* `4` version, development was shifted back to MATLAB.
+Heiko Schütt implemented Bayesian inference based on numerical integration rather than MCMC
+sampling, helped by Stefan Harmeling and Felix Wichmann as well as statistical
+support by Jakob Macke. Numerical integration avoids the reliability issues
+plaguing *psignifit* `3`. Furthermore, *psignifit* `4` is fast enough to be used without
+MEX files. These two changesi–avoiding the pain of MCMC chain diagnosis and
```

Comment:
typo: "changesi"

Reply:
fixed in b140de2c8b41c410127b72b7725b158df293a3a0